sdk/communication/AzureCommunicationChat/Tests/ChatThreadClientTests.swift

```diff
@@ -592,7 +592,7 @@ class ChatThreadClientTests: XCTestCase {
     }
 
     func test_ListParticipants_ReturnsParticipants() {
-        let user2 = settings.user2!
```

Comment:
Why the change to the force unwrap when the previous version wasn't optional? What benefits does settings gain from being made optional?

Reply:
A recordable test isn't required to have settings--we just provide a mechanism for them to do so.